Test.Microsoft.NET.Build.Containers.Filesystem/TargetsTests.cs

```diff
@@ -37,7 +37,7 @@ public static void Cleanup()
         if (CombinedTargetsLocation != null) File.Delete(CombinedTargetsLocation);
     }
 
-    private Project InitProject(Dictionary<string, string> bonusProps)
+    private (Project, IDisposable) InitProject(Dictionary<string, string> bonusProps, string logFileName = "log")
```

Comment:
the disposable return is added here because it allows loggers to flush themselves. without it, configured binaryloggers would never flush to their backing files.